app/controllers/coronavirus_form/product_details_controller.rb

```diff
@@ -47,9 +47,10 @@ def selected_made_in_uk?
 
   def validate_fields(product)
     missing_fields = validate_missing_fields(product)
+    product_equipment_type = selected_ppe? ? validate_radio_field("#{PAGE}.equipment_type", radio: product["equipment_type"]) : []
```

Comment:
I don't like this line – please suggest something nicer 😁

Reply:
Or maybe something for later

Reply:
I think if it works lets ship it, I've not liked most of the PRs I've put up recently 😄I'm hoping we'll be given time for improving the codebase post-launch!